Backpack/NavigationBar/Classes/BPKNavigationBarLargeTitleView.m

```diff
@@ -0,0 +1,62 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#import "BPKNavigationBarLargeTitleView.h"
+
+#import <Backpack/Label.h>
+#import <Backpack/Spacing.h>
+
+const CGFloat BPKNavigationBarLargeTitleViewHeight = 52;
```

Comment:
Should this be hard-coded? Surely the height should be based on constraints and the `contentSize` of the title

Reply:
No, it's the other way around because we want the title label to get clipped when the view decreases in size. If we base the height of the view on the height of the label then the label shrinks as the view height decreases which isn't the intended effect